dlog/plonk-15-wires/src/prover.rs

```diff
@@ -333,18 +336,18 @@ where
             println!("p{} f_comm {:?}", line!(), index.srs.get_ref().commit_non_hiding(&f, None));
         } */
 
-        let f = &(&(&(&(&(&index.cs.gnrc_lnrz(&e[0].w)
+        let f = &(&(&(&(&(&index.cs.gnrc_lnrz(&evals[0].w)
```

Comment:
Would aid readability to add a comment with the equation and description above this

Reply:
yeah I think readability of the prover/verifier could be improved :o I've started listing ideas of future improvements we could work on here: https://github.com/o1-labs/proof-systems/issues/135